.github/workflows/cherry-pick-rc-to-develop.yml

```diff
@@ -0,0 +1,63 @@
+name: "Cherry-pick from rc to develop"
+
+on:
+    pull_request:
+        branches:
+            - release/candidate
+        types:
+            - closed
+
+jobs:
+    cherry-pick:
+        runs-on: ubuntu-latest
+        if: github.event.pull_request.merged == true
+
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+
+            - name: Append -cherry-pick to branch name
+              id: extract
+              run: |
+                  PR_BRANCH="${{ github.event.pull_request.head.ref }}"
+                  NEW_BRANCH_NAME="${PR_BRANCH}-cherry-pick"
+                  echo "New branch name: $NEW_BRANCH_NAME"
+                  echo "::set-output name=newBranchName::$NEW_BRANCH_NAME"
+
+            - name: Check if changes only in kalium submodule
+              id: check_changes
+              run: |
+                  NUM_CHANGES=$(git diff origin/develop --name-only | grep -v '^kalium/' | wc -l)
```

Comment:
👨🏼‍🍳

Reply:
Not needed in kalium :D 

I'm removing it.